source/tyr/tests/integration/datapost_test.py

```diff
@@ -0,0 +1,37 @@
+from tests.check_utils import api_get, api_post, api_delete, api_put, _dt
+import json
+import pytest
+import os
+from navitiacommon import models
+from tyr import app
+import logging
+
+@pytest.fixture(scope="module")
+def create_instance_fr():
+    with app.app_context():
+        instance = models.Instance('fr')
+        models.db.session.add(instance)
+        models.db.session.commit()
+        return instance.id
+
+def test_post_pbf(create_instance_fr):
+    filename = 'empty_pbf.osm.pbf'
+    path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'tests/fixtures/', filename)
+    files = {'file': (open(path, 'rb'), filename)}
```

Comment:
the close of the file handler is added (i don't know how to check the flask's tester behaviour)